src/utils/Footer/Footer.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+enum Filter {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+type Props = {
+  completedTodos: number;
+  notCompletedTodos: number;
+  filter: string;
+  setFilter: (filter: string) => void;
+  handleClearCompleted: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  completedTodos,
+  notCompletedTodos,
+  filter,
+  setFilter,
+  handleClearCompleted,
+}) => {
+  const filterLinks = [
+    { key: Filter.All, label: 'All', href: '#/' },
+    { key: Filter.Active, label: 'Active', href: '#/active' },
+    { key: Filter.Completed, label: 'Completed', href: '#/completed' },
+  ];
```

Comment:
Consider changing it with Enum or move it to out from component better to constant file

Reply:
it's a constant, so better to move it out of the component
<img width="665" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/70220667/84d01472-70c6-4049-992f-b45e000d9b38">